src/App.jsx

```diff
@@ -25,10 +25,14 @@ const App = () => {
     onMount();
   }, []);
 
+  const onLogout = () => {
+    setUser(false)
+  }
+
   if (user === null) return <Loader />;
 
   return (
-    <AppContext.Provider value={{ user }}>
+    <AppContext.Provider value={{ user, onLogout }}>
```

Comment:
Really good,  I would pass setUser straight to the Logout component, and there call setUser(false). This way we make sure we don't create an extra onLogout function. But I guess its also a matter of preference.

Reply:
Yes, I changed it afterwards when I got to exercise 3 😅